src/components/Modal.tsx

```diff
@@ -25,22 +25,18 @@ const Modal = ({ selector, size, prevent }: ModalProps) => {
       size={size}
       actions="static"
     >
-      <h5 className="tds-modal-headline" slot="tds-modal-headline">
+      <h5 className="tds-modal-headline" slot="headler">
```

Comment:
```suggestion
      <h5 className="tds-modal-headline" slot="header">
```

Reply:
Ah, NVM just realised this is the implementation, that ship has sailed for now ⛵️

Reply:
Also it makes more sense with heading when its the implementation

Reply:
Apparently we aren't fully aligned on the header vs heading and subheader vs subheading naming for our slots, we have both but in different components.

Reply:
Where did you notice things are off?